Warehouse.Infrastructure/ProcutsRepository/ProductRemoteDataRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Warehouse.Application.Interfaces.Repositories;
+using Warehouse.Domain.Entities;
+using Warehouse.Infrastructure.RestClient.Configuration;
+using Warehouse.Infrastructure.RestClient.Products;
+
+
+namespace Warehouse.Infrastructure
+{
+	public class ProductRemoteDataRepository : IProductRepository
+    {
+        private readonly MockyProductsConfiguration _configuration;
```

Comment:
This Mocky configuration should be injected in HttpClientService

Reply:
Done.